ios/tools/KeyboardCalibrator/KeyboardCalibrator/DummyInputViewController.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  DummyInputViewController.swift
+//  DefaultKeyboardHost
+//
+//  Created by Joshua Horton on 1/13/20.
+//  Copyright © 2020 Joshua Horton. All rights reserved.
```

Comment:
Technically copyright needs to be with SIL. Important but not important y'know ;-)

Reply:
Gotcha.  Good ol' Xcode-autocomments I didn't think to adjust.